crates/compiler/parse/src/pattern.rs

```diff
@@ -551,7 +551,10 @@ fn record_pattern_field<'a>() -> impl Parser<'a, Loc<Pattern<'a>>, PRecord<'a>>
         // (This is true in both literals and types.)
         let (_, opt_loc_val, state) = optional(either(
             byte(b':', PRecord::Colon),
-            byte(b'?', PRecord::Optional),
+            and(
+                byte(b'?', PRecord::Optional),
+                optional(byte(b'?', PRecord::Optional)),
```

Comment:
Same here, we probably want `OptionalFirstQuestionMark` or `OptionalSecondQuestionMark`

Reply:
Doen